drivers/base/bootvid/common.c

```diff
@@ -272,6 +325,21 @@ RleBitBlt(
 
 /* PUBLIC FUNCTIONS ***********************************************************/
 
+/*
+ * @implemented
+ */
+ULONG
+NTAPI
+VidSetTextColor(IN ULONG Color)
```

Comment:
Will do. Some of functions really lack good documentation, like what input values are expected, and what output format is needed.

Reply:
It would be good to change the bootvid header https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/drivers/bootvid/bootvid.h;h=23971957f4d4e48f34879dfab4e492eeed1fa0e1;hb=refs/heads/master

Reply:
Updated header :)